packages/python/plotly/plotly/tests/test_core/test_px/test_px.py

```diff
@@ -232,3 +233,9 @@ def assert_orderings(days_order, days_check, times_order, times_check):
     for days in permutations(df["day"].unique()):
         for times in permutations(df["time"].unique()):
             assert_orderings(days, days, times, times)
+
+
+def check_permissive_defaults():
```

Comment:
I think your function should be called `test_something` so that pytest catches it in the test suite (otherwise it will consider it a helper function). You can verify this by executing locally pytest in your machine (for example in the same directory, `pytest -v test_px.py` and check whether the new function appears). Maybe I'm wrong but please check :-).

Reply:
You're right about tests needing to start with `test` in order to be picked up by `pytest`.

After modifying to above code, the output of running `pytest -v test_px.py` is:

```
test_px.py::test_scatter PASSED
test_px.py::test_custom_data_scatter PASSED
test_px.py::test_labels PASSED
test_px.py::test_px_templates PASSED
test_px.py::test_orthogonal_orderings PASSED
test_px.py::test_permissive_defaults PASSED

Reply:
awesome!